tests/matlab_test_data_collectors/python_testers/makeCartCircle_test.py

```diff
@@ -17,16 +17,24 @@ def test_makeCartCircle():
         filepath = os.path.join(collected_values_folder, f'{i:06d}.mat')
         recorded_data = loadmat(filepath)
 
-        radius, num_points, center, arc_angle = recorded_data['params'][0]
-        center = Vector(center[0])
-        num_points = num_points[0][0]
-        radius = radius[0][0]
-        arc_angle = arc_angle[0][0]
-        if np.isclose(arc_angle, 2 * np.pi):
-            arc_angle = 2 * np.pi
+        if len(recorded_data['params'][0]) == 4:
+            radius, num_points, center, arc_angle = recorded_data['params'][0]
+            center = Vector(center[0])
+            num_points = num_points[0][0]
+            radius = radius[0][0]
+            arc_angle = arc_angle[0][0]
+            if np.isclose(arc_angle, 2 * np.pi):
+                arc_angle = 2 * np.pi
+
+            circle = make_cart_circle(radius, num_points, center, arc_angle)
+        else:
+            radius, num_points, center = recorded_data['params'][0]
```

Comment:
Maybe assign `recorded_data['params'][0]` to a variable? It is used multiple times.

Reply:
done!